src/app/content/content.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from "@angular/core";
```

Comment:
do we need this component? isn't it better just add `<app-toolbar (toggle)="toggleSideBar()"></app-toolbar>` and `<div class="content"></div>` (which will be changed by "transactions component") directly in `<mat-sidenav-content>`?